src/quo/components/tabs/tabs/view.cljs

```diff
@@ -180,6 +177,15 @@
                                   :fade-end?           fade-end?
                                   :fading              fading
                                   :on-scroll           on-scroll})
+             :on-layout (fn []
+                          (js/setTimeout (fn []
+                                           (.scrollToIndex @flat-list-ref
+                                                           #js
+                                                            {:animated false
+                                                             :index    (utils.collection/first-index
+                                                                        #(= @active-tab-id (:id %))
+                                                                        data)}))
+                                         1))
```

Comment:
Alternatively, I'm using the `scrollToIndex` to set the initial scroll position. Without `setTimeout`, this approach fails about 20% of the time. As far as I've tested with the timeout, it works reliably.